vllm/executor/ray_gpu_executor.py

```diff
@@ -265,9 +263,9 @@ def _run_workers(
         self,
         method: str,
         *args,
-        driver_args: Optional[Tuple[Any]] = None,
+        driver_args: Optional[Tuple[Any, ...]] = None,
         driver_kwargs: Optional[Dict[str, Any]] = None,
-        all_args: Optional[List[List[Any]]] = None,
+        all_args: Optional[List[Tuple[Any, ...]]] = None,
         all_kwargs: Optional[List[Dict[str, Any]]] = None,
         use_dummy_driver: bool = False,
         max_concurrent_workers: Optional[int] = None,
```

Comment:
BTW, it's better to document in the doc string to note several usages:

1. only args/kwargs , all workers share the same args
2. only args/kwargs + driver_args/driver_kwargs , all workers except for the driver share the same args
3. all_args/all_kwargs , specify each arg/kwargs for each worker

`use_dummy_driver` is orthogonal to the above, indicating whether to use dummy driver worker.

As this function goes more and more complicated, I'm thinking of simplifying the usecases. Maybe only 1 and 3 are allowed.